src/components/ComponentPreview.stories.tsx

```diff
@@ -1330,3 +1331,26 @@ export const LeafletMap: Story = {
     await expect(await canvas.findByRole('button', {name: 'Zoom out'})).toBeVisible();
   },
 };
+
+export const Partners: Story = {
+  name: 'Partners',
+  args: {
+    component: {
+      id: 'wekruya',
+      type: 'partners',
+      key: 'partners',
+      label: 'A partners preview',
+      tooltip: 'An example for the tooltip',
+      description: 'A description for the Partners component',
+      defaultValue: [
+        {
+          bsn: '123456788',
+          initials: 'J',
+          affixes: 'Van',
+          lastName: 'Deniston',
+          dateOfBirth: '2000-09-08',
+        },
+      ],
```

Comment:
```suggestion
      defaultValue: [],
```
I think the preview should make sense even when "nothing" is configured on the component.

We make similar pragmatic choices like that in the columns and edit grid component where we just render some placeholder data instead of "real" children/default values.